katello/backup.rb

```diff
@@ -0,0 +1,423 @@
+require 'optparse'
+require 'fileutils'
+require 'date'
+require 'yaml'
+require 'find'
+require 'json'
+require 'highline/import'
+require "/usr/share/katello/helper.rb"
```

Comment:
This makes it impossible to test the code from a git checkout :(

How about

```ruby
require File.join(File.expand_path('..', __FILE__), 'helper.rb')
```

instead? :)

Reply:
@evgeni I put [instructions](https://git.io/vdklj) here a few times on how to check this out for testing. Those will be a little out of date, but I also tried to update it in newer comments (that may be hidden by github). 

Will the above solution place the files in `/usr/share/katello` in a production install? That is where we agreed to put the utility class files

Reply:
The above does not move the files anywhere, it just requires them with a relative path.
We even could just use require_relative here in most cases.

Reply:
@evgeni how does that work with the rpm install?